tests/ui/pages/desktop/base.py

```diff
@@ -26,13 +35,70 @@ def login(self, email, password):
     def logout(self):
         self.header.click_logout()
 
+    @property
+    def amo_logo_title(self):
+        return self.selenium.find_element(*self._amo_logo_link_locator).get_attribute('title')
+
+    def get_url_current_page(self):
+        time.sleep(1)
+        # This is a hack until selenium actually learns patience
```

Comment:
Can you cite the issue here? We don't want to litter our tests with arbitrary sleeps, and I'd prefer not to wrap the Selenium method for obtaining the URL in our base page object.